src/bma250e/javaupm_bma250e.i

```diff
@@ -1,10 +1,25 @@
 %module javaupm_bma250e
 %include "../upm.i"
 %include "typemaps.i"
-%include "../upm_vectortypes.i"
+%include "../upm_stdvector.i"
 
 %ignore getAccelerometer(float *, float *, float *);
 
+%typemap(javaimports) SWIGTYPE %{
+import java.util.AbstractList;
+import java.lang.Float;
+%}
+
+%typemap(javaout) upm::BMA250E {
+    return new $&javaclassname($jnicall, true);
+}
+%typemap(javaout) std::vector<float> {
+    return (AbstractList<Float>)(new $&javaclassname($jnicall, true));
+}
+%typemap(jstype) std::vector<float> "AbstractList<Float>"
+
+%template(floatVector) std::vector<float>;
+
```

Comment:
Hi, we only added them for Float types as that sensor only has functions returning std::vector\<float\>. We only added what was needed (not that it would make a difference if the other would be added)

As for adding these in a separate file and including them wherever needed might prove problematic for sensors that have std::vector\<\> as both input and output (see * explanation for javaupm_bno055.i).


But we could add these in a separate file and include them wherever needed (to keep those files cleaner), and treat special sensors (bno055) in their own \.i file.